kpi/tests/test_mock_data_exports.py

```diff
@@ -678,33 +678,10 @@ def test_anotheruser_can_export_when_submissions_publicly_shared(self):
 
         anonymous_user = get_anonymous_user()
 
-        assert self.asset.has_perm(self.anotheruser, PERM_VIEW_ASSET) == False
-        assert PERM_VIEW_ASSET not in self.asset.get_perms(self.anotheruser)
-        assert self.asset.has_perm(self.anotheruser, PERM_CHANGE_ASSET) == False
-        assert PERM_CHANGE_ASSET not in self.asset.get_perms(self.anotheruser)
-
+        assert not self.asset.has_perm(self.anotheruser, PERM_VIEW_SUBMISSIONS)
         # required to export
-        self.asset.assign_perm(self.anotheruser, PERM_CHANGE_ASSET)
```

Comment:
if we're still testing #2870, as the docstring indicates, has the scalpel been too aggressive here? wouldn't we still need to give `anotheruser` some explicit permission that isn't  `view_submissions`?